src/Microsoft.DocAsCode.Build.Engine/TemplateProcessors/TemplateModelTransformer.cs

```diff
@@ -116,6 +116,7 @@ internal ManifestItem Transform(InternalManifestItem item)
                             message = $"Error transforming model generated from \"{item.LocalPathFromRoot}\" using \"{template.ScriptName}\". To get the detailed raw model, please run docfx with debug mode --debug. {e.Message} ";
                         }
 
+                        Logger.LogError(message, code: ErrorCodes.Build.TemplateModelTransformError);
```

Comment:
According to [template system](https://dotnet.github.io/docfx/tutorial/intro_template.html), 2 steps are named `Renderer` and `Preprocessor`. We can use `ApplyTemplateRendererError`, and `ApplyTemplatePreprocessorError` for below one.